classroom.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_students(student)
+    @student.push(student)
```

Comment:
- [ ] You seem to have made a syntax error here. The student parameter cannot be pushed here because you defined **`@students`** array in `initialize`, not `@student`. Kindly fix this.